### ISSUE TITLE
'usage' does not reflect current state.

### ISSUE THREAD

**Issue by fennecdjay** [desyncr/zpm-zsh#10]
for now, it just prints:

```sh
Usage:
  zpm 'zsh-users/zsh-syntax-highlighting'
```
but I think it could add:
```sh
  zpm reset
  zpm list
  zpm update
```
What do you think? am I forgetting something?
I can do it and submit a pull request if you like.

**Comment by desyncr** [comment]
Sounds good to me. May be we could work on a `help` command:

    zpm help

    Usage:
        zpm 'zsh-users/zsh-syntax-highlighting'
 
    Available commands:
        zpm reset
        zpm list
        zpm update


When the user issues `zpm` without arguments, the help message should be displayed.

What do you think? If you're up for it please go ahead!

**Comment by fennecdjay** [comment]
That seems good!
I'm not very comfortable with **git** yet, so I'll probably wait until we're done with #9 to do this.

**Comment by desyncr** [comment]
@fennecdjay Awesome! Looking forward to this!

**Comment by fennecdjay** [comment]
Nice. see #12. (#11 was a mistake, in my opinion, as it ships #9 changes).

**Comment by fennecdjay** [comment]
You migth close this, as it seems #12 solves it.

**Comment by desyncr** [comment]
Merged #12
